share/spack/gitlab/ascent_pipeline.yml

```diff
@@ -1,6 +1,8 @@
 merge_pipeline:
   only:
   - develop
+  variables:
+    SPACK_REF: ${CI_COMMIT_SHA}
```

Comment:
I guess just one thing to check is that the downstream `e4s` pipeline is actually using this variable name, and not something else like `SPACK_REFERENCE` or something.